azure-pipelines.yml

```diff
@@ -194,7 +194,7 @@ stages:
         timeoutInMinutes: 90
         pool:
           name: Azure Pipelines
-          vmImage: macos-latest
+          image: macos-14-arm64
```

Comment:
Was copying from https://github.com/microsoft/sbom-tool/blob/20f43605fa20db121f86de73366284970c22f0a9/pipelines/sbom-tool-pr-build.yaml#L55-L57 but it's not working. 😕